checks/load.go

```diff
@@ -21,6 +21,12 @@ var availableChecks = map[string]ArgableFunc{
 		args["interpreter"] = "/bin/bash"
 		return RunScript{}.FromArgs(args)
 	},
+	"yum-installed": func(args map[string]interface{}) (Checker, error) {
+		return YumInstalled{}.FromArgs(args)
+	},
+	"apt-installed": func(args map[string]interface{}) (Checker, error) {
```

Comment:
The old name was dpkg-installed, should probably add another one here with that just for completeness.